io.github.mak448a.QTCord.yml

```diff
@@ -0,0 +1,23 @@
+app-id: io.github.mak448a.QTCord
+runtime: org.kde.Platform
+runtime-version: '5.15-23.08'
+sdk: org.kde.Sdk
+command: io.github.mak448a.QTCord
+modules:
+  - python3-modules.json
+  - name: BuildQTCord
+    buildsystem: simple
+    build-commands:
+      - install -D src/flatpak_builder/io.github.mak448a.QTCord /app/bin/io.github.mak448a.QTCord
+      - mkdir -p /app/share/applications/
+      - install -Dm 644 src/smiley.svg ${FLATPAK_DEST}/share/icons/hicolor/scalable/apps/${FLATPAK_ID}.svg
+      - mv src/io.github.mak448a.QTCord.desktop /app/share/applications/io.github.mak448a.QTCord.desktop
+      - mv src/* /app/bin
+    sources:
+      - type: git
+        url: https://github.com/mak448a/QTCord
+        tag: 0.0.0
+        commit: 05b112b22d3d7fa6daea79a2c2666ff403795130
+finish-args:
+  --socket=wayland
+  --share=network
```

Comment:
missing `--socket=fallback-x11`, and `--share=ipc`

Reply:
Ok adding